e2e/portalicious/tests/DoPayment/DoSuccessfulPaymentWithExcelFsp.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { test } from '@playwright/test';
+import { format } from 'date-fns';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import NLRCProgramPV from '@121-service/src/seed-data/program/program-nlrc-pv.json';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationsPvExcel } from '@121-service/test/registrations/pagination/pagination-data';
+
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import PaymentsPage from '@121-e2e/portalicious/pages/PaymentsPage';
+import RegistrationsPage from '@121-e2e/portalicious/pages/RegistrationsPage';
+
+// Export Excel FSP payment list
+const amount = NLRCProgramPV.fixedTransferValue;
+const fullName = registrationsPvExcel[0].fullName;
+const addressStreet = registrationsPvExcel[0].addressStreet;
+const addressHouseNumber = registrationsPvExcel[0].addressHouseNumber;
+const addressPostalCode = registrationsPvExcel[0].addressPostalCode;
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdPV = 2;
+  const PVProgramId = programIdPV;
```

Comment:
This is another candidate of the same import-magic trick. ;)

Reply:
Had to be really hidden ;)